### DISCUSSION TITLE
Concerned about the plan...

### DISCUSSION THREAD

**Discussion by pacemaker82** [springfall2008/batpred#734] [General]
I have had a good experience with PredBat for a few days but today its being really odd. The plan has it holding at 100% until 1500 tomorrow (its 20:15 now), and its charged to 100% in the latest half hour (from 90% something) which is way more expensive than just waiting a few hours, using the battery for free, and charging at cheaper rates.

Instead my plan says to run from grid (or solar) until 1500 tomorrow and keep at 100%. That seems like an odd choice. Any thoughts or places I can look? My config is pretty much OOTB I dont mess with it.
<img width="464" alt="Screenshot 2024-02-06 at 20 11 13" src="https://github.com/springfall2008/batpred/assets/157808562/2784cc9f-5949-40dc-8046-de7502812aba">
<img width="467" alt="Screenshot 2024-02-06 at 20 11 21" src="https://github.com/springfall2008/batpred/assets/157808562/2058d8e6-ad9e-4c30-85e9-2a8b9d576680">

**Comment by springfall2008** [comment]
It's probably not quite as bad a plan as you expect as using the grid avoids the losses.

That said try setting metric battery cycle to 1 or 0 and see what you get then

**Comment by pacemaker82** [comment]
I set it to 1, the plan has changed to running in eco mode effectively during day this time, with solar expected to charge. 

I didnt find anything in the docs about metric_battery_cycle - what did it do?

Edit - scrap that, search didnt match on it, but I found it otherwise...

**Reply by springfall2008** [reply]
It's here in the docs https://springfall2008.github.io/batpred/customisation/#battery-loss-options

**Reply by pacemaker82** [reply]
thanks. Any reason why it would choose the 20p slot to charge now rather than just waiting for cheaper? Especially at 90% SOC at the end of the evening, the predicted usage is low - that seems like the most odd choice
